src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/armor/RadiationTask.java

```diff
@@ -28,13 +33,20 @@
  */
 public class RadiationTask extends AbstractArmorTask {
 
+    public static final int GRACE_PERIOD_DURATION = 15;
```

Comment:
lets config option this

Reply:
I did think on this, and I'm not sure I see the direct benefit of adding it as a config option. Given the usage here it seems to fit more as a direct constant, not unlike the other behaviours surrounding radioactivity as a whole?

Reply:
people generally prefer to have a lot of values configurable.
Thats why i suggested it

Reply:
is there an existing mc/bukkit config value we can use? i know there's a general grace period in mc.

Reply:
Added config